client/laststand.lua

```diff
@@ -95,15 +95,15 @@ function EndLastStand()
     TriggerServerEvent("hospital:server:SetLaststandStatus", false)
 end
 
--- Events
-
 ---@param bool boolean
-RegisterNetEvent('hospital:client:SetEscortingState', function(bool)
+---TODO: this event name should be changed within qb-policejob to be generic
+AddEventHandler('hospital:client:SetEscortingState', function(bool)
     isEscorting = bool
 end)
 
 ---@param bool boolean
-RegisterNetEvent('hospital:client:isEscorted', function(bool)
+---TODO: this event name should be changed within qb-policejob to be generic
```

Comment:
I think you can do an `@`todo here

Reply:
That doesn't seem to be a supported annotation https://github.com/sumneko/lua-language-server/wiki/Annotations#documenting-types